src/portal/urls.py

```diff
@@ -0,0 +1,16 @@
+from django.conf.urls import include, url
+from django.contrib.auth import views as auth_views
+from . import views
+
+urlpatterns = [
+    url(r'', include('social_django.urls', namespace='social')),
+    url(r'^$', views.index, name='home'),
+    url(r'^login/$',
```

Comment:
I am confused how to do it exactly. Based on http://stackoverflow.com/questions/22775697/how-to-specify-long-url-patterns-using-regex-so-that-they-follow-pep8-guidelines.